_maps/map_files/generic/z2.dmm

```diff
@@ -94,28 +94,24 @@
 /turf/open/floor/holofloor/plating,
 /area/holodeck/rec_center/offline)
 "as" = (
-/turf/open/floor/holofloor{
```

Comment:
As in, otherwise people can use the holodeck to farm floor materials 'might not want to'